frontend/app/src/types/history/tx.ts

```diff
@@ -83,6 +83,13 @@ export interface AddressesAndEvmChainPayload {
   readonly evmChain: string;
 }
 
+export const EvmChainAddress = z.object({
+  address: z.string(),
+  evmChain: z.string()
+});
+
+export type EvmChainAddress = z.infer<typeof EvmChainAddress>;
```

Comment:
I will merge it as it is but we should maybe move the type somewhere else in the future as it is not only related to transactions